ecs.tf

```diff
@@ -161,6 +165,10 @@ resource "aws_ecs_task_definition" "airflow" {
         "secrets": [{
           "name": "FERNET_KEY",
           "valueFrom": "arn:aws:ssm:${var.region}:${data.aws_caller_identity.this.account_id}:parameter/${var.resource_suffix}/airflow/fernet_key"
+        },
+        {
+          "name": "AIRFLOW__CORE__FERNET_KEY",
+          "valueFrom": "arn:aws:ssm:${var.region}:${data.aws_caller_identity.this.account_id}:parameter/${var.resource_suffix}/airflow/fernet_key"
```

Comment:
Pretty sure this is invalid and that ValueFrom only works with secrets.

Reply:
This is under the secrets, it's a couple of lines above.

Reply:
Ah now I see it.

Reply:
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html Airflow can be configured using these `AIRFLOW__CORE` environment variables. 

I'm just trying to get the fernet key to be established in all of the places it might be required. The variables in airflow keep getting invalidated when docker restarts because it's unable to read the variables from the database.